File: app/src/main/java/com/tekzee/amiggos/ui/memories/ourmemories/custom/CustomLoadingItem.java
package com.tekzee.amiggos.ui.memories.ourmemories.custom;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.recyclerview.widget.RecyclerView;

import com.tekzee.amiggos.R;

import ru.alexbykov.nopaginate.item.LoadingItem;

public class CustomLoadingItem implements LoadingItem {
   
        @Override
        public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_loading_item, parent, false);
            return new RecyclerView.ViewHolder(view) {
            };
        }
        @Override
        public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
   
        }
   
}

File: app/src/main/java/com/tekzee/amiggoss/ui/mylifestyle/custom/CustomLoadingItem.kt
package com.tekzee.amiggoss.ui.mylifestyle.custom

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.tekzee.amiggoss.R
import ru.alexbykov.nopaginate.item.LoadingItem

class CustomLoadingItem : LoadingItem {
    override fun onCreateViewHolder(
        parent: ViewGroup,
        viewType: Int
    ): RecyclerView.ViewHolder {
        val view = LayoutInflater.from(parent.context)
            .inflate(R.layout.item_loading, parent, false)
        return object : RecyclerView.ViewHolder(view) {}
    }

    override fun onBindViewHolder(
        holder: RecyclerView.ViewHolder,
        position: Int
    ) {
    }
}